sysutils/pkg/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        freebsd pkg 1.10.4
+categories          sysutils archivers
+platforms           darwin
+license             BSD
+maintainers         {@apowers313 ato.ms:apowers} \
+                    openmaintainer
+description         Package management tool for FreeBSD
+long_description    Pkg is the Next Generation package management tool for FreeBSD. Its main goals are to faciliate remote binary package upgrades. It also works with ports without remote binary packages.
+homepage            https://wiki.freebsd.org/pkgng
+
+depends_lib         port:libarchive \
+                    port:openssl
```

Comment:
Not sure what the current state of discussion w.r.t. libressl vs. openssl is, but we usually write dependencies on OpenSSL as `path:libssl.dylib:openssl` so that people running libressl can use that instead.

Reply:
fixed in 7f695b2